docker-compose.integration.yml

```diff
@@ -1,23 +1,29 @@
 version: '3.4'
-volumes:
-  node_modules:
+
+networks:
+  mojaloop-net:
+    name: mojaloop-net
+
+#volumes:
+#  node_modules:
 
 services:
-  redis:
```

Comment:
Hmm...how is this working when removed when the scheme adapter depends on it?

Reply:
this file is deleted